docs/remove-ads.md

```diff
@@ -112,15 +115,11 @@ window.addEventListener('load', () => {
 })
 ```
 
-As soon as the page loads, any visitor who does not have Web Monetization
-(`!document.monetization`) sees ads immediately.
-
-If the visitor _does_ have Web Moentization we start our three second timer.
-When the timer is up we check whether the `monetizationstart` event has fired.
-If Web Monetization hasn't initialized, we show them the ads.
+If the visitor _does_ have Web Monetization, the three-second timer starts.
```

Comment:
I would put this with the previous paragraph before the code block, because both apply to the same block of code. Or we could split the code block into

```js
window.addEventListener('load', () => {
  if (!document.monetization) {
    showAds()
```
and
```js
  } else {
    setTimeout(() => {
      if (!hasPaid) {
        showAds()
      }
    }, 3000)
  }
})
```

Reply:
Fixed